fakestorage/server.go

```diff
@@ -291,6 +301,71 @@ func (s *Server) Client() *storage.Client {
 	return client
 }
 
+func (s *Server) handleBatchCall(w http.ResponseWriter, r *http.Request) {
+	reader, err := r.MultipartReader()
+	if err != nil {
+		http.Error(w, "invalid Content-Type header", http.StatusBadRequest)
+		return
+	}
+
+	mw := multipart.NewWriter(w)
+	w.Header().Set("Content-Type", "multipart/mixed; boundary="+mw.Boundary())
+
+	w.WriteHeader(http.StatusOK)
+	part, err := reader.NextPart()
+	for ; err == nil; part, err = reader.NextPart() {
+		contentId := part.Header.Get("Content-ID")
+		if contentId == "" {
+			// missing content ID, skip
+			continue
+		}
```

Comment:
nit: can you rename this to `contentID`?

```suggestion
		contentID := part.Header.Get("Content-ID")
		if contentID == "" {
			// missing content ID, skip
			continue
		}
```